server/routes/prisons/exclusionDates/index.ts

```diff
@@ -1,16 +1,19 @@
 import { RequestHandler, Router } from 'express'
-import { Services } from '../../../services'
+import { Services, SessionTemplateService } from '../../../services'
```

Comment:
This `SessionTemplateService` import isn't needed. It is why your build has failed the lint check on CircleCi: https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-visits-internal-admin-ui/346/workflows/8d3a7b09-1faa-4b1d-b931-54b3b1267743/jobs/1811?invite=true#step-107-9